tokens/tsm-report.go

```diff
@@ -0,0 +1,37 @@
+package tokens
+
+import (
+	"encoding/json"
+
+	"github.com/fxamacker/cbor/v2"
+)
+
+const TSMReportMediaType = "application/vnd.veraison.tsm-report+cbor"
+
+type TSMReport struct {
+	AuxBlob       []byte           `json:"auxblob,omitempty"`
+	OutBlob       []byte           `json:"outblob"`
+	Provider      string           `json:"provider"`
+	ServiceReport TsmServiceReport `json:"service,omitempty"`
```

Comment:
In the CDDL, `service-report` is a group, not a map.

The way to model it is by embedding the relevant fields:

```suggestion
	ManifestBlob    BinaryString `json:"manifestblob,omitempty"`
	ServiceProvider *string      `json:"service_provider,omitempty"`
```

rather than creating a nested structure.